BaseCollector.py

```diff
@@ -249,12 +249,12 @@ def generate_metrics_enriched_by_api(self, no_match_in_config: list, label_names
             gauges[new_metric_suffix].add_metric(labels=labels, value=value)
         return gauges
 
-    def generate_alert_metrics(self, label_names: list) -> GaugeMetricFamily:
+    def generate_alert_metrics(self, label_names: list) -> InfoMetricFamily:
         if 'alert_name' not in label_names:
             label_names.extend(['alert_name', 'alert_level', 'status', 'alert_impact'])
-        gauge = GaugeMetricFamily(f'vrops_{self.vrops_entity_name}_alert', 'vrops-exporter',
-                                  labels=label_names)
-        return gauge
+        alert_metric = InfoMetricFamily(f'vrops_{self.vrops_entity_name}_alert', 'vrops-exporter',
```

Comment:
So we're completely moving to Info here? Can you elaborate? We would need to change `describe` then too, since it is still Gauge.

Reply:
This eliminates the need to access the protected `_labelnames` attribute. The `InfoMetricFamily` can accept prepared dicts with `label name` - `label value` pairs. With this we can easily set the individual alert labels. 

Describe needs to be changed, right.